<ui version="4.0" >
 <class>Solid</class>
 <widget class="QWidget" name="Solid" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>General option</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7" >
      <item>
       <spacer name="horizontalSpacer_10" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>138</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>maximum particle compaction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCompaction" />
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_11" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>137</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Field label</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <spacer name="horizontalSpacer_7" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>264</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="comboBoxField" >
        <property name="minimumSize" >
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_8" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>263</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxInteractions" >
     <property name="title" >
      <string>Particles interactions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" >
      <item>
       <spacer name="horizontalSpacer_5" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>138</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>friction model</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxFriction" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>granular model</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxGranular" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>kinetic model</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxKinetic" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLabel" name="labelCoupling" >
            <property name="text" >
             <string>polydispersed : coupling between dispersed phases</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxCoupling" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_6" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>137</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>641</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
